rest/results-tabulation-api/ExtendedTallySheetVersion_PE_R1.py

```diff
@@ -1,93 +0,0 @@
-from flask import render_template
-from ext.ExtendedTallySheet import ExtendedTallySheetVersion
-from orm.entities import Area
-from constants.VOTE_TYPES import Postal
-from util import to_comma_seperated_num
-from orm.enums import AreaTypeEnum
-
-
-class ExtendedTallySheetVersion_PE_R1(ExtendedTallySheetVersion):
-
-    def html_letter(self, title="", total_registered_voters=None):
-        return super(ExtendedTallySheetVersion_PE_R1, self).html_letter(
-            title="Results of Polling Division %s" % self.tallySheetVersion.submission.area.areaName,
-            total_registered_voters=float(get_polling_division_total_registered_voters(tallySheetVersion=self))
-        )
-
-    def html(self, title="", total_registered_voters=None):
-        tallySheetVersion = self.tallySheetVersion
```

Comment:
@callkalpa This `html` method content logic must be places inside the `html_letter` method of PE_CE_RO_V1